.circleci/config.yml

```diff
@@ -238,40 +283,45 @@ jobs:
             pip3.8 install conbench
             pip3.8 install git+https://github.com/conbench/benchalerts.git@0.3.1
             pip3.8 install scripts/veloxbench
+      - run-benchmarks
+      # TODO: Disabling conbench github notifications for now until results are more stable.
```

Comment:
nit: Lets get rid of the whole comment..

Reply:
We've got the code in history anyway, so that's fine. We could keep the TODO and drop the code too